test/Driver/multiple_swiftmodule_inputs.swift

```diff
@@ -0,0 +1,100 @@
+// RUN: %swiftc_driver -c -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver %s single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -c -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver -c %s
+// RUN: %swiftc_driver multiple_swiftmodule_inputs.o single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver -emit-module %s
+// RUN: %swiftc_driver multiple_swiftmodule_inputs.swiftmodule single_int.swiftmodule
+
+// RUN: %swiftc_driver -c -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver %s single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -c -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver -c %s
+// RUN: %swiftc_driver multiple_swiftmodule_inputs.o single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -emit-module-path single_int.swiftmodule -module-name single_int -parse-as-library %S/Inputs/single_int.swift
+// RUN: %swiftc_driver -emit-module %s
+// RUN: %swiftc_driver multiple_swiftmodule_inputs.swiftmodule single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -### -emit-module -module-name single_int -parse-as-library %S/Inputs/single_int.swift 2>&1 | %FileCheck %s -check-prefix=EMIT-MODULE
+// RUN: %swiftc_driver -### -g -emit-module -module-name single_int -parse-as-library %S/Inputs/single_int.swift 2>&1 | %FileCheck %s -check-prefix=EMIT-MODULE
+// EMIT-MODULE: bin/swift
+// EMIT-MODULE: -emit-module
+// EMIT-MODULE: -primary-file {{.*}}/single_int.swift
+// EMIT-MODULE: -emit-module-doc-path {{.*}}/single_int-{{.*}}.swiftmodule
+// EMIT-MODULE: -parse-as-library
+// EMIT-MODULE: -module-name single_int
+// EMIT-MODULE: -o single_int.swiftmodule
+
+
+// RUN: %swiftc_driver -### multiple_swiftmodule_inputs.o single_int.swiftmodule 2>&1 | %FileCheck %s -check-prefix=LINK-OBJECT-AND-MODULE
```

Comment:
Please don't rely on the host platform being Darwin; use a target triple here too. `-target x86_64-apple-macosx10.10` is a pretty reasonable one.